tonic/src/status.rs

```diff
@@ -3,6 +3,7 @@ use crate::metadata::MetadataMap;
 use bytes::Bytes;
 use http::header::{HeaderMap, HeaderValue};
 use percent_encoding::{percent_decode, percent_encode, AsciiSet, CONTROLS};
+use std::convert::TryFrom;
```

Comment:
Looks like this is failing CI?

Reply:
Looking.

Reply:
Fixed.

Reply:
It seems like `deny(warnings)` is only set in CI? When I run `cargo clippy`, it only warned for this and some other issues. It would probably be useful to make the crates enable `#![deny(warnings)]` (but in a separate PR obviously).